pkg/types/baremetal/platform.go

```diff
@@ -6,9 +6,10 @@ import (
 
 // BMC stores the information about a baremetal host's management controller.
 type BMC struct {
-	Username string `json:"username"`
-	Password string `json:"password"`
-	Address  string `json:"address"`
+	Username                       string `json:"username"`
+	Password                       string `json:"password"`
+	Address                        string `json:"address"`
+	DisableCertificateVerification bool   `json:"disable_certificate_verification"`
```

Comment:
Oh sorry missed this on my first pass: platform config json fields should be camel case:

```suggestion
	DisableCertificateVerification bool   `json:"disableCertificateVerification"`
```

Reply:
Done